Sources/SocksCore/TCPSocket.swift

```diff
@@ -39,22 +39,29 @@ public protocol TCPReadableSocket: TCPSocket { }
 extension TCPReadableSocket {
 
     public func recv(maxBytes: Int = BufferCapacity) throws -> [UInt8] {
+    	var out = Array<UInt8>()
```

Comment:
rm (also, spacing is off on this. doesn't matter since we're removing it, but if it stays, please align properly)